civictechprojects/static/css/partials/_IconLinkDisplay.scss

```diff
@@ -25,14 +25,18 @@
 
 .IconLink-right {
   float: right;
-  width: 200px;
+  width: 170px;
   height: 64px;
   line-height: 23px;
   border-top: 1px solid $color-grey-medium;
   border-top-right-radius: 5px;
   border-right: 1px solid $color-grey-medium;
   border-bottom-right-radius: 5px;
   border-bottom: 1px solid $color-grey-medium;
+  display: block;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
```

Comment:
I think this works for now, let's call it good. Further fixes to IconLink seem best in another PR because they show up on multiple pages so I don't want them to be blocking. If you want to roll up a full iconlink fix in this PR though let me know, and I can push a commit or something.

Reply:
This currently works on all the existing pages.  We can do more comprehensive fixes in a separate PR.